Code-Conventions.md

```diff
@@ -13,7 +13,77 @@ The following are some common mistakes not covered by the code style check:
 
 ## IntelliJ Integration
 
-The easiest way to follow our style is to use IntelliJ's formatter. To install it, simply download [this xml](http://ix.io/1jNP) and [follow this guide](https://www.jetbrains.com/help/idea/configuring-code-style.html#import-code-style) to import it into your IDE. Then you can press `Ctrl+Alt+L` to format the current document.
+The easiest way to follow our style is to use IntelliJ's formatter. To install it, [follow this guide](https://www.jetbrains.com/help/idea/configuring-code-style.html#import-code-style) to import the xml listed below into your IDE. Once installed, you can press `Ctrl+Alt+L` to format the current document.
```

Comment:
```suggestion
The easiest way to follow our style is to use IntelliJ's formatter. To install it, copy and save the xml listed below, then [follow this guide](https://www.jetbrains.com/help/idea/configuring-code-style.html#import-code-style) to import it into your IDE. Once installed, you can press `Ctrl+Alt+L` to format the current document.
```

I don't want to make a release on a wiki just for this and NFC isn't attached to it being an automatic download, so I'll just re-word it this way